Author: Otto Kekäläinen <otto@seravo.fi>
Description: Add existing man pages to CMake to actually include them in pacakge

--- old/man/CMakeLists.txt	2014-06-18 14:02:56.249087477 +0300
+++ new/man/CMakeLists.txt	2014-06-18 14:04:21.901091309 +0300
@@ -14,6 +14,7 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
 
 SET(MAN1_SERVER innochecksum.1 my_print_defaults.1 myisam_ftdump.1 myisamchk.1
+		aria_chk.1 aria_dump_log.1 aria_ftdump.1 aria_pack.1 aria_read_log.1
 		myisamlog.1 myisampack.1 mysql.server.1
 		mysql_convert_table_format.1 mysql_fix_extensions.1
 		mysql_install_db.1
